package/tailscale-cert-renewal.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Tailscale Certificate Renewal Timer
+Requires=tailscale-cert-renewal.service
+
+[Timer]
+OnCalendar=weekly
+Persistent=true
```

Comment:
Any reason you haven't specified the `Unit=tailscale-cert-renewal.service` here (in the same way we're doing in the `tailscale-install.timer`)?